docs/concepts/gui-components.mdx

```diff
@@ -0,0 +1,243 @@
+---
+title: "GUI Components"
+description: "Customize how your agents appear and behave in the BeeAI platform interface"
+---
+
+GUI components in the BeeAI platform define how your agents appear and behave in the user interface. Through the `PlatformUIAnnotation` metadata, you can customize the visual presentation and interaction patterns for your agents.
```

Comment:
I'd add word graphical here so there's no confusion with CLI UI which does not support this yet.

```suggestion
GUI components in the BeeAI platform define how your agents appear and behave in the graphical user interface. Through the `PlatformUIAnnotation` metadata, you can customize the visual presentation and interaction patterns for your agents.
```